types/vimeo__player/vimeo__player-tests.ts

```diff
@@ -484,6 +508,10 @@ player.on('bufferend', (data) => {
    // no associated data with this event
 });
 
+player.on('seeking', (data) => {
```

Comment:
This is incorrect: the `seeking` event [does return data](https://github.com/vimeo/player.js/#seeking):

```
{
    duration: number
    percent: number
    seconds: number
}
```

Should add `console.log` for test to inspect the returned payload;

```
console.log(data.duration);
console.log(data.percent);
console.log(data.second);
```